esmvaltool/recipes/clouds/recipe_bock24acp_fig6_zonal.yml

```diff
@@ -0,0 +1,579 @@
+  - {dataset: BNU-ESM, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: CanESM2, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: CCSM4, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: FGOALS-g2, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: GFDL-CM3, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: MPI-ESM-LR, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: MPI-ESM-MR, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+
+DATASETS_ECS_LOW: &datasets_ecs_low
+  # CMIP6
+  - {dataset: CAMS-CSM1-0, grid: gn}
+  - {dataset: GISS-E2-1-G, grid: gn}
+  - {dataset: MIROC6, grid: gn}
+  - {dataset: MIROC-ES2L, ensemble: r1i1p1f2, grid: gn}
+  - {dataset: NorESM2-LM, grid: gn, institute: NCC}
+  - {dataset: NorESM2-MM, grid: gn, institute: NCC}
+  # CMIP5
+  - {dataset: bcc-csm1-1, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: bcc-csm1-1-m, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: GFDL-ESM2G, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: GFDL-ESM2M, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: GISS-E2-H, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: GISS-E2-R, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: inmcm4, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: IPSL-CM5B-LR, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: MIROC5, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: MRI-CGCM3, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: NorESM1-M, ensemble: r1i1p1, project: CMIP5}
+
+DATASETS_ECS_LOW_scenario: &datasets_ecs_low_scenario
+  # CMIP6
+  - {dataset: CAMS-CSM1-0, grid: gn, exp: ssp585, end_year: 2099}
+  - {dataset: GISS-E2-1-G, grid: gn, ensemble: r1i1p1f2, exp: ssp585}
+  - {dataset: MIROC6, grid: gn, exp: ssp585}
+  - {dataset: MIROC-ES2L, ensemble: r1i1p1f2, grid: gn, exp: ssp585}
+  - {dataset: NorESM2-LM, grid: gn, institute: NCC, exp: ssp585}
+  - {dataset: NorESM2-MM, grid: gn, institute: NCC, exp: ssp585}
+  # CMIP5
+  - {dataset: bcc-csm1-1, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: bcc-csm1-1-m, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: GFDL-ESM2G, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: GFDL-ESM2M, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: GISS-E2-H, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: GISS-E2-R, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: inmcm4, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: IPSL-CM5B-LR, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: MIROC5, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: MRI-CGCM3, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+  - {dataset: NorESM1-M, ensemble: r1i1p1, project: CMIP5, exp: rcp85}
+
+
+preprocessors:
+
+  zonal_mmm:
+    custom_order: true
+    regrid:
+      target_grid: 2x2
+      scheme: linear
+    zonal_statistics:
+      operator: mean
+    multi_model_statistics:
+      span: full
+      statistics: [mean, p5, p95]
```

Comment:
The syntax `p5` and `p95` for percentiles [has been deprecated in version v2.10.0 and will be removed in v2.12.0](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/changelog.html#id8). See https://github.com/ESMValGroup/ESMValCore/pull/2553 how to use the new syntax.

This also needs to be changed in the diagnostic, see e.g., https://github.com/ESMValGroup/ESMValTool/blob/configurable_mpl_monitor/esmvaltool/diag_scripts/examples/make_plot.py#L45.

Reply:
Thanks, @schlunma for spotting this!
I realised that I don't use this preprocessor block. Since I have different model groups, I need to calculate the percentiles in the diag_script. I removed this part now from the recipe.